app/models/thesis.rb

```diff
@@ -96,7 +96,7 @@ class Thesis < ApplicationRecord
   #  includes(:user).order('users.surname, users.given_name')
   # }
   scope :date_asc, -> { order('grad_date') }
-  scope :in_review, -> { where('publication_status == ?', 'Publication review') }
+  scope :in_review, -> { where('publication_status = ?', 'Publication review') }
```

Comment:
D'oh!

Reply:
lol yeah it's funny that sqlite was fine with this whereas Postgres immediately noped out of it